lib/Format.hs

```diff
@@ -1543,6 +1544,24 @@ lambda log span indentation indent' lambda' = case lambda' of
       <> sourceToken log indent' blank arrow
       <> exprPrefix log span indentation indent' expr'
 
+getSymbol :: Language.PureScript.CST.LetBinding a -> CST.Ident
+getSymbol (Language.PureScript.CST.LetBindingName _ valueBindingFields') = CST.nameValue (CST.valName valueBindingFields')
+getSymbol (Language.PureScript.CST.LetBindingPattern span binder' equals where'') = undefined
```

Comment:
Still need to clarify what does this case represent

Reply:
ok, looks like it's for destructuring-like bindings e.g.
```haskell
let
  Pair a b = x
  y:ys = arg
```
basically bindings that don't start with a name